config/prod.toml

```diff
@@ -52,6 +52,8 @@
             maxHostIdleConns        = 1000
             responseHeaderTimeoutMs = 25000
             tlsHandshakeTimeoutMs   = 2000
+    [worker.schedulerconfig]
+            nodeBindingRefreshIntervalMins = 60
```

Comment:
Weren't we keeping it 15 mins?

Reply:
I thought we decided with 1 hour only. Will change it if need be